File: platform/processor/live-insight/src/test/testData/loops/FunctionLoops.java
public class FunctionLoops {
    private void function() {
        for (int i = 0; i < 10; i++) {
            System.out.println(i);
        }
        for (Object o : new Object[10]) {
            System.out.println(o);
        }
        while (true) {
            System.out.println("while");
        }
        do {
            System.out.println("do");
        } while (true);
    }
}

File: platform/processor/live-insight/src/test/testData/kotlin/loops/FunctionLoops.kt
class FunctionLoops {
    private fun function() {
        for (i in 0..9) {
            System.out.println(i)
        }
        for (o in arrayOfNulls<Object>(10)) {
            System.out.println(o)
        }
        while (true) {
            System.out.println("while")
        }
        do {
            System.out.println("do")
        } while (true)
    }
}